src/test/unit/process-profile.test.js

```diff
@@ -51,7 +51,8 @@ describe('extract functions and resource from location strings', function() {
     ] = extractFuncsAndResourcesFromFrameLocations(
       locationIndexes,
       stringTable,
-      libs
+      libs,
+      []
```

Comment:
I feel like we should add some mock data to this to make sure we have coverage of your new code. Also this is a snapshot test, so you should be able to run `yarn test -u` (I usually do `jest -u`, but I'm not sure how paths are set up on other people's machines.)